Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:10.9.0-alpine as builder
```

Comment:
Would a `.nvmrc` be useful in this repository?

Reply:
I don't know. When I look at other NPM project, I can't find `.nvmrc` file in them. We have a safe-guard with the `engine` attribute of the `package.json`